tests/utils.py

```diff
@@ -43,9 +43,10 @@
 FILE_ORIENTATION_U = TEST_FILES_DIR / "orientation_u.fastq"
 FILE_2000_RECORDS = TEST_FILES_DIR / "2000_records.fastq"
 FILE_SINGLE = TEST_FILES_DIR / "single.fastq"
+FILE_SOURCE_FRUIT_FLY = TEST_FILES_DIR / "fruit_fly.fastq"
 FILE_SRA_SAMPLE_1 = TEST_FILES_DIR / "sra_sample_1.fastq"
 FILE_SRA_SAMPLE_2 = TEST_FILES_DIR / "sra_sample_2.fastq"
-FILE_TRANSCRIPTS = TEST_FILES_DIR / "transcripts.fa"
+FILE_TRANSCRIPTS = TEST_FILES_DIR / "transcripts.fasta"
```

Comment:
I'm confused. It seems you added a new file `transcripts.fasta`. But there was (and still _is_) already a file `transcripts.fa`. How do they differ?

If they are both valid but functionally different (i.e. they are used to test different behavior), then rename them so that it's clear what they are. If they are the same, just keep one. If the old one is invalid, also just keep one.